pkg/shares/share_splitting_test.go

```diff
@@ -64,7 +64,7 @@ func TestSplitTxs(t *testing.T) {
 				append([]uint8{
 					0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, // namespace id
 					0x0, // info byte
-					0x0, // reserved byte
+					0xb, // BUG: reserved byte should be zero
```

Comment:
[question]

for my own understanding,  this change to a different bug by removing the forced 0, and then we will fix this in an upcoming PR for #817

Reply:
Yep, exactly!